managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedLedgerImpl.java

```diff
@@ -248,6 +248,7 @@ public void operationComplete(ManagedLedgerInfo mlInfo, Stat stat) {
                     if (log.isDebugEnabled()) {
                         log.debug("[{}] Opening legder {}", name, id);
                     }
+                    store.recordRead();
```

Comment:
Yes, they need to be separate, since they have very different profiles : 
 * Reads are served from memory directly by the ZK server we are connected to
 * Writes needs to be routed through the ZK leader and be accepted by the majority of the quorum, including syncing on the local disk.